scripts/conf/go/Dockerfile

```diff
@@ -26,6 +22,6 @@ RUN apt-get -y update \
 
 COPY . /go/src/github.com/ulule/loukoum
 WORKDIR /go/src/github.com/ulule/loukoum
```

Comment:
You can change the workdir to something like `/media/ulule/loukoum` to remove the usage of `GO111MODULE` as env variable.